proxy-kubernetes/argocd/apps/seichi-debug-gateway/bungeecord/deployment.yaml

```
@@ -0,0 +1,128 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: bungeecord
+  name: bungeecord
+  namespace: seichi-debug-gateway
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: bungeecord
+  template:
+    metadata:
+      labels:
+        app: bungeecord
+    spec:
+      containers:
+      - command:
+        - sh
+        - -c
+        - eval "echo \"$(cat /plugins/RedisBungee/config.yml)\"" > /plugins/RedisBungee/config.yml;
+          /usr/bin/run-bungeecord.sh
+        env:
+        - name: MEMORY
+          value: 768m
+        - name: TYPE
+          value: WATERFALL
+        image: gcr.io/manifest-shell-237511/seichi-debug-gw-bungeecord:latest
+        name: bungeecord
+        ports:
+        - containerPort: 25577
+          name: minecraft
+        readinessProbe:
+          failureThreshold: 2
+          initialDelaySeconds: 15
+          periodSeconds: 5
+          successThreshold: 2
+          tcpSocket:
+            port: 25577
+          timeoutSeconds: 1
+        resources:
+          requests:
+            memory: 768Mi
+      imagePullSecrets:
+      - name: seichi-network-gcr-private-image
```

```suggestion
      - name: gcr-bungeecord-image-pull-secret
```

https://github.com/GiganticMinecraft/seichi_infra/blob/913263754873449c86488d96e4cf74c50eba6ff5/terraform/lke_cluster_secrets.tf#L72